data-prepper-expression/src/main/java/org/opensearch/dataprepper/expression/OperatorProvider.java

```diff
@@ -35,4 +35,8 @@ public OperatorProvider(final List<Operator<?>> operators) {
         }
         return operator;
     }
+
+    public boolean containsOperatorSymbol(final int symbol) {
```

Comment:
As we already have `getOperator`, should this be have a different name? Perhaps: `containsOperator` or `supportsOperator`?